app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<p style="color: green"><%= notice %></p>
+
+<div>
+
+    <h2 class=" text-center">Recipe <%= @recipe.id%></h2>
+    <div class="row">
+    <div class="col-md-6">
+      <p>Preparation Time: <%= @recipe.preparation_time %></p>
+    </div>
+    <div class="col-md-6">
+      <div class="d-flex justify-content-end">
+        <p>public: </p>
+        <% if @recipe.public %>
+          <%= image_tag('icon-true.svg', size: '30x30') %>
+        <% else %>
+          <%= image_tag('toggle-off-solid.svg', size: '30x30') %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+    <p>Cooking Time: <%= @recipe.cooking_time%></p>
+    <div class="ingredients">
+    <%= link_to "Add Ingredients", edit_recipe_path(@recipe)%>
+      <a href= <%= "/general_shopping_list" %>>
+        <button class="btn-bg-red-big ">Generate Shopping List</button>
```

Comment:
- [ ] Great job implementing some required changes. This required changes from the previous reviewer is yet to be implemented. The Add ingredients button should lead to the form that allows the user to add new food, the button helps the user to add new food 'ingredients' that will display in the table below the two buttons not to edit the recipe and the ingredients in the table will help us to generate the shopping list by doing the sum of the ingredients on the list and displaying it on the shopping list page like shown in the wireframe. Kindly fix this issue.

### App Behaviours
[add-ingrea.webm](https://github.com/IsmailChangezi/recipe-app-rails/assets/26597432/84457d66-0a7e-49aa-be3a-c9fb96804888)